src/utils/token-verifier.ts

```diff
@@ -43,6 +44,8 @@ export const ID_TOKEN_INFO: FirebaseTokenInfo = {
   jwtName: 'Firebase ID token',
   shortName: 'ID token',
   expiredErrorCode: AuthClientErrorCode.ID_TOKEN_EXPIRED,
+  errorCodeType: AuthClientErrorCode,
```

Comment:
ID_TOKEN_INFO and SESSION_COOKIE_INFO should be defined in the auth module.

Reply:
Good call. Moved all auth related items to a new `token-verifier-util.ts`